server/lomas_server/tests/test_mongodb_admin.py

```diff
@@ -526,6 +646,77 @@ def test_del_dataset(self) -> None:
         with self.assertRaises(ValueError):
             del_dataset(self.db, dataset)
 
+    def test_show_dataset(self) -> None:
+        """Test show dataset"""
+        with self.assertRaises(ValueError):
+            dataset_found = show_dataset(self.db, "PENGUIN")
+
+        dataset = "PENGUIN"
+        database_type = PrivateDatabaseType.PATH
+        dataset_path = "some_path"
+        metadata_database_type = PrivateDatabaseType.PATH
+        metadata_path = "./tests/test_data/metadata/penguin_metadata.yaml"
+
+        add_dataset(
+            self.db,
+            dataset,
+            database_type,
+            metadata_database_type,
+            dataset_path=dataset_path,
+            metadata_path=metadata_path,
+        )
+        dataset_found = show_dataset(self.db, "PENGUIN")
```

Comment:
Should we rename the "show" functions to "get"?

Reply:
yes, good idea

Reply:
i started but it is such a mess with the notebooks, documentation and all.. I will juste create a ticket and there will a PR just for that to keep it simpler